### PR TITLE
Improve messages

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
Improve messages when migrating WPSdk and FrameworkVersion.

### CODE REVIEW

**Comment by teobugslayer** [comment]
:+1:

**Comment by endarova** [comment]
:+1:

File: lib/project.ts [hunk 1]
```diff
@@ -297,12 +297,12 @@ export class Project implements Project.IProject {
...
 					}
 
-					var shouldUpdateFramework = this.$prompter.confirm(util.format("You are trying to use version of Windows Phone SDK that is not supported for your project's Cordova version. Do you want to use %s?", selectedFramework.DisplayName)).wait()
+					var shouldUpdateFramework = this.$prompter.confirm(util.format("You cannot build with the Windows Phone %s SDK with the currently selected target version of Apache Cordova. Do you want to switch to %s?", newVersion, selectedFramework.DisplayName)).wait()
 					if(shouldUpdateFramework) {
 						this.onFrameworkVersionChanging(selectedFramework.Version).wait();
 						this.projectData.FrameworkVersion = selectedFramework.Version;
 					} else {
-						this.$errors.failWithoutHelp("Unable to use Windows Phone SDK %s as the current Cordova version %s does not support it. You should target at least Cordova 3.7.0 in order to use Windows Phone 8.1 SDK.", newVersion, selectedFramework.Version);
+						this.$errors.failWithoutHelp("Unable to set Windows Phone %s as the target SDK. Migrate to Apache Cordova 3.7.0 and try again.", newVersion);
```
Comment (ikoevska) [inline lib/project.ts]: Apache Cordova 3.7.0 or later

**Comment by ikoevska** [comment]
:+1:

**Pull request merged**